### PR TITLE
[CMPN-22] Allow all HTML options on input in PbSelectableIcon, PbSelectableCard and PbSelectableCardIcon Rails kits

### PR DESCRIPTION
**Pull request by josephinef9**
#### Runway Link

https://nitro.powerhrg.com/runway/backlog_items/CMPN-22 

#### Screens
Selectable card: 

Screen Shot 2020-09-29 at 8 49 10 AM
Screen Shot 2020-09-24 at 4 27 35 PM

Selectable Icon: 

Screen Shot 2020-09-29 at 8 49 37 AM
Screen Shot 2020-09-24 at 4 26 59 PM

Selectable Card Icon: 

Screen Shot 2020-09-29 at 8 49 22 AM
Screen Shot 2020-09-24 at 4 25 46 PM
 

#### Breaking Changes

[No (Explain)] - Only additive 

#### How to test this
Rspec tests have been written to cover changes. 

#### Checklist:

- [x] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `depreciation`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [x] **URGENCY** Please select a release milestone
- [x] **DEPLOY** Please add the `Milano` label when you are ready for a review.
- [x] **SCREENSHOT** Please add a screen shot or two.
- [x] **SPECS** Please cover your changes with specs.
- [x] **READ DOCS** Please make sure you have read and understand the [Playbook Release Process](https://github.com/powerhome/playbook/wiki/Playbook-Releases)

*The normal release cut off deadline is 3p EDT each week. Please reach out to the release team if you have an urgent request or need a release off cycle.*

### CODE REVIEW

File: app/pb_kits/playbook/pb_selectable_card/_selectable_card.html.erb [hunk 1]
```diff
@@ -1,12 +1,12 @@
-<%= content_tag(:div,
+<%= content_tag(:label,
```
Comment (garettarrowood) [inline app/pb_kits/playbook/pb_selectable_card/_selectable_card.html.erb:1]: Did you mean to leave this change? Can you change it back or provide some explanation?

File: app/pb_kits/playbook/pb_selectable_card/docs/example.yml [hunk 2]
```diff
@@ -1,11 +1,12 @@
 examples:
-  
+
   rails:
   - selectable_card_default: Default
   - selectable_card_single_select: Single Select
   - selectable_card_block: Block
+  - selectable_card_options: Options
```
Comment (garettarrowood) [inline app/pb_kits/playbook/pb_selectable_card/docs/example.yml:7]: Can we title all 3 of these doc additions "With Options" instead?

File: app/pb_kits/playbook/pb_selectable_card/selectable_card.rb [hunk 3]
```diff
@@ -8,21 +8,21 @@ class SelectableCard
...
-                  default: false
+                      default: false
       prop :icon, type: Playbook::Props::Boolean,
                   default: false
       prop :multi, type: Playbook::Props::Boolean,
-                  default: true
+                   default: true
       prop :input_id, type: Playbook::Props::String
+
+      prop :additional_input_options, type: Playbook::Props::Hash,
```
Comment (garettarrowood) [inline app/pb_kits/playbook/pb_selectable_card/selectable_card.rb:20]: After talking with UX, to keep consistency across all kits with inputs, let's call this prop simply `input_options` instead. That means the two `input_options` methods you have in two of the kits here would be better renamed to `additional_input_options` (switch out the naming).

**Review by garettarrowood — CHANGES_REQUESTED** [review]
This looks great! See the comments above for some minor changes.

**Review by garettarrowood — APPROVED** [review]

**Review by jasoncypret — APPROVED** [review]

**Review by jasperfurniss — APPROVED** [review]

**Pull request merged**